### ISSUE TITLE
Align the search icon within the search bar

### ISSUE THREAD

**Issue by UMNIYAH** [cheton/styled-ui#873]
After migrating from v1 to v2, the search icon within the search bar is not centred vertically, which affects the visual balance and may impact user experience.

Proposed Label: 🐛 bug

v1 vs. v2
![Screenshot 2024-06-12 at 9 39 09 AM](https://github.com/trendmicro-frontend/tonic-ui/assets/76800891/1284ddad-ef0a-4b7a-b5dd-a9b13d279acf)
![Screenshot 2024-06-12 at 9 38 46 AM](https://github.com/trendmicro-frontend/tonic-ui/assets/76800891/9d8ece7f-0098-4cf1-9fe6-02bf52e29c74)

**Comment by cheton** [comment]
I noticed that the key difference is the missing `vertical-align: middle` style property on the `svg` element. I will add it back to the `SVGIcon` component in the `@tonic-ui/react-icons` package.

**Comment by UMNIYAH** [comment]
> I noticed that the key difference is the missing `vertical-align: middle` style property on the `svg` element. I will add it back to the `SVGIcon` component in the `@tonic-ui/react-icons` package.
 
 Thank you (ﾉ◕ヮ◕)ﾉ*:･ﾟ✧

**Comment by cheton** [comment]
This issue was resolved in [@tonic-ui/react@2.0.2](https://github.com/trendmicro-frontend/tonic-ui/releases/tag/%40tonic-ui%2Freact%402.0.2). You can update both packages to 2.0.2.

```
"@tonic-ui/react": "^2.0.2",
"@tonic-ui/react-icons": "^2.0.2",
```

**Comment by UMNIYAH** [comment]
> This issue was resolved in [@tonic-ui/react@2.0.2](https://github.com/trendmicro-frontend/tonic-ui/releases/tag/%40tonic-ui%2Freact%402.0.2). You can update both packages to 2.0.2.
> 

Thanks a bunch for the quick reply!
